notes-20200211.tex

```diff
@@ -129,6 +129,12 @@ \subsection{(Usual) Equality Types}
 
 In this sense, equality types and judgmental equality are equivalent. Formally, the rules of equality types consist of those of identification types and the equality reflection rule.
 
+Note: One implementation of equality types additionally has the following uniqueness rule:
```

Comment:
Sorry for not being clear. I meant it might be better to change "and the equality reflection rule" to "and the equality reflection rule and uniqueness" (a very strong UIP for equality types). All implementations of equality types should have this rule or make it derivable.